experimental/regression_suite/shark-test-suite-models/sdxl/test_scheduler.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+from ireers_tools import *
+import os
+from conftest import VmfbManager
+from pathlib import Path
+
+rocm_chip = os.getenv("ROCM_CHIP", default="gfx90a")
```

Comment:
Is `ROCM_CHIP` always set in the builders? I think in test_unet.py is defaulted to gfx942, is there any reason we use gfx90a instead?

Reply:
Yeah it's always set as part of the workflow file, so runners always get gfx942. This default is more for end users. I just changed all defaults to gfx942, so it's consistent though